python/lust.py

```diff
@@ -29,6 +29,24 @@ def __calculate_factorial(self, argument):
       result *= i
     return result
 
+class SquareCommand(LustObject):
```

Comment:
Please document what are you doing here!

Reply:
necu!